crates/meroctl/src/cli/context/identity.rs

```diff
@@ -1,12 +1,15 @@
+use calimero_context_config::types::Capability as ConfigCapability;
 use calimero_primitives::alias::Alias;
 use calimero_primitives::context::ContextId;
 use calimero_primitives::identity::PublicKey;
 use calimero_server_primitives::admin::GetContextIdentitiesResponse;
-use clap::Parser;
+use clap::{Parser, ValueEnum};
 use eyre::{OptionExt, Result as EyreResult, WrapErr};
+use grant::GrantPermissionCommand;
 use libp2p::identity::Keypair;
 use libp2p::Multiaddr;
 use reqwest::Client;
+use revoke::RevokePermissionCommand;
```

Comment:
these should be after line 25, since they're imports from a submodule, not an external crate

Reply:
@miraclx Alright, done.